doc/nrf-bm/libraries/ble_radio_notification.rst

```diff
@@ -0,0 +1,55 @@
+.. _lib_ble_radio_notification:
+
+Bluetooth: Radio Notification
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Bluetooth Low Energy® Radio Notification library allows the user to subscribe to the Radio Notification signal to receive notification prior to and after radio events.
+
+Overview
+********
+
+The library allows the user to register a handler to receive the radio notifications and configure the distance in microseconds between the active Radio Notification signal and the radio event.
+
+Configuration
+*************
+
+The library is enabled and configured entirely using the Kconfig system.
+Set the :kconfig:option:`CONFIG_BLE_RADIO_NOTIFICATION` Kconfig option to enable the library.
+
+The library uses the ``SWI02`` IRQ.
+Use the :kconfig:option:`CONFIG_BLE_RADIO_NOTIFICATION_IRQ_PRIO` Kconfig option to set the IRQ priority.
+The library can be configured to receive notifications before the radio is active, after the radio is active or both before and after.
+This can be chosen by setting the :kconfig:option:`CONFIG_BLE_RADIO_NOTIFICATION_ON_ACTIVE`, :kconfig:option:`CONFIG_BLE_RADIO_NOTIFICATION_ON_INACTIVE` or :kconfig:option:`CONFIG_BLE_RADIO_NOTIFICATION_ON_BOTH` Kconfig option, respectively.
```

Comment:
```suggestion
This can be chosen by setting the :kconfig:option:`CONFIG_BLE_RADIO_NOTIFICATION_ON_ACTIVE`, the :kconfig:option:`CONFIG_BLE_RADIO_NOTIFICATION_ON_INACTIVE`, or the :kconfig:option:`CONFIG_BLE_RADIO_NOTIFICATION_ON_BOTH` Kconfig option, respectively.
```